docs/installation/keycloak/keycloak-minikube.md

```diff
@@ -0,0 +1,178 @@
+# How to Set Up Headlamp in a Minikube Cluster with Keycloak OIDC Authentication
+
+In this tutorial, we'll walk through the process of configuring Headlamp within a Minikube cluster while utilizing Keycloak for OIDC (OpenID Connect) authentication. This tutorial is based on Keycloak version 22.0.4, Minikube version v1.31.2, and Headlamp version 0.20.1.
+
+## Creating a Keycloak Client
+
+To enable OIDC authentication in your Minikube cluster, you'll need to create a Keycloak client. Follow these steps to configure the client in your Keycloak admin panel:
+
+1. Start by accessing your Keycloak admin panel.
+![Keycloak Admin Panel](./keycloak-admin-panel.jpeg)
+
+2. Navigate to the "Clients" section andClick on the "Create client" option.
```

Comment:
missing a space `Navigate to the "Clients" section andClick on the "Create client" option.` to `Navigate to the "Clients" section and click on the "Create client" option.`